test/integration_test.rb

```diff
@@ -44,6 +44,27 @@ def test_tests_are_selected_based_on_changes_in_current_working_tree
         output = rake('ttnt:test:run')
         assert_match '2 runs, 2 assertions, 2 failures', output[:stdout]
       end
+
+      def test_isolated
+        # Make TTNT select all tests
+        git_rm_and_commit("#{@repo.workdir}/.ttnt", 'Remove .ttnt')
+        ENV['ISOLATED'] = '1'
+        output = rake('ttnt:test:run')
+        ENV.delete('ISOLATED')
```

Comment:
That's right :bulb: Thank you!
Fixed in this commit: https://github.com/Genki-S/ttnt/commit/ad15fed7aa218c1157b4653fa3df1cab71fbf979